Source/Terrain/Private/TerrainActor.cpp

```diff
@@ -26,35 +28,49 @@ ATerrainActor::ATerrainActor()
 	m_TerrainMeshComponent = CreateDefaultSubobject<URuntimeMeshComponent>(TEXT("Terrain0"));
 	m_TerrainMeshComponent->SetCollisionProfileName(UCollisionProfile::BlockAll_ProfileName);
 	m_TerrainMeshComponent->Mobility = EComponentMobility::Static;
-	m_TerrainMeshComponent->SetMeshSectionCollisionEnabled(0, true);
+	//m_TerrainMeshComponent->SetMeshSectionCollisionEnabled(0, true);
```

Comment:
did you test collision on the mesh? can you trace to it and/or can UCharacter walk on it?